src/components/binding/binding-panel.js

```diff
@@ -215,13 +215,20 @@ genotet.BindingPanel.prototype.updateFileListAfterLoading = function() {
     this.selectGenes_[index] = select;
     select.val(track.fileName).trigger('change');
 
-    // Set track fileName
-    select.on('select2:select', function(event) {
-      var fileName = event.params.data.id;
-      this.signal('updateTrack', {
-        trackIndex: index,
-        fileName: fileName
-      });
-    }.bind(this));
+    // Set track fileName and highlight the track hovered in panel.
+    select
+      .on('select2:select', function(event) {
+        var fileName = event.params.data.id;
+        this.signal('updateTrack', {
+          trackIndex: index,
+          fileName: fileName
+        });
+      }.bind(this))
+      .off().on('select2:open', function(event) {
```

Comment:
"because we only want to trigger the highlighting when we open the binding file list select2" -> false. I think the requirement is highlight on hovering select list. The highlighting is for the user to observe which track corresponds to which select. Clicking is one more unnecessary perceptional gap.